File: CarpoolSharing.API/Models/Ride.cs
using System;
using System.Collections.Generic;

namespace CarpoolSharing.API.Models
{
    public class Ride
    {
        public int RideId { get; set; }
        public string StartLocation { get; set; }
        public string EndLocation { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public ICollection<EmployeeRide> EmployeeRides { get; set; }
        public Car Car { get; set; }
        public int CarId { get; set; }
    }
}

File: CarpoolSharing-SPA/src/app/_models/ride.ts
import { Employee } from './employee';
import { Car } from './car';

export interface Ride {
    rideId?: number;
    startLocation?: string;
    endLocation?: string;
    employeeRides?: Employee[];
    employees?: Employee[];
    carPlates?: string;
    car?: Car;
    startDate?: string;
    endDate?: string;
    carId?: number;
}
